RevenueCatUI/PaywallView.swift

```diff
@@ -181,7 +184,7 @@ public struct PaywallView: View {
         self.content
             .displayError(self.$error, dismissOnClose: true)
             .preference(key: PaywallDidLoadPreferenceKey.self,
-                        value: self.offering != nil && self.customerInfo != nil)
```

Comment:
this value is not correct,
we've added `Purchases.shared.attribution.isThereUnsyncedAttributes` - but this was left out.
I opted to use a separate flag for it instead.